ci/svd2rust-regress/src/main.rs

```diff
@@ -133,8 +175,9 @@ fn main() {
     tests.par_iter().for_each(|t| {
         let start = Instant::now();
 
-        match svd_test::test(t, &bin_path) {
+        match svd_test::test(t, &bin_path, rustfmt_bin_path) {
             Ok(()) => {
+                // TODO: If verbosity is > 1, print every logged stderr
```

Comment:
TODO check

Reply:
This one I feel can be implemented later.